docs/Community/governance/sig-charter-template.md

```diff
@@ -0,0 +1,82 @@
+SIG YOURSIG Charter
+===================
+
+## Scope
+
+Include a two to three sentence summary of the work this SIG will do. Imagine
+trying to explain it to someone who knows what Open Enclave SDK does but isn't
+yet familiar with all the internals.
+
+## In Scope
+
+### Code, Binaries, or Services
+
+- list of what qualifies as a deliverable (code, binary, or service)
+- that is within the scope of this SIG
+- for example, driver code for specific hardware,
+- or documentation .md files,
+- but DO NOT list specific code paths or locations here
+
+### Cross-cutting or Externally Facing Processes
+
+- list of processes owned by this SIG, such as
+- the creation of a periodic release .tar.gz file,
+- or the ongoing operation of CI/CD infrastructure,
+- or the management of a mentorship program.
+
+### Out of Scope
+
+List some things that might seem like they fall in-scope for this SIG, but don't (or don't yet).
+
+## Process & Management
+
+THIS SIG meets at least THIS OFTEN.
+
+Meetings are held at PLACE, with AGENDA/MINUTES available or not. Meetings are OPEN/CLOSED.
+
+Membership in the SIG may be gained by PROCESS.
+
+The SIG Chair is APPOINTED/ELECTED for TIME PERIOD.
+
+## Roles & Structure
+
+### Chair
+
+- Role: Chair of a SIG or WG
+- Responsibilities
```

Comment:
For SIGs, are the responsibilities cumulative as you move from Reviewer to Approver to Chair? I would assume so, but the chair responsibilities seem pretty distinct from the other two.

Reply:
I think that's up to the community // that SIG to decide.

in my experience, the privileges are cumulative but the responsibilities are not. For instance, a chair of a busy SIG may not have time to do as many reviews as a reviewer, since the chair's role is more organizational and less tactical.